internal/clients/team4/client.go

```diff
@@ -21,14 +21,18 @@ func init() {
 func newClientInternal(clientID shared.ClientID, testing *testing.T) client {
 	// have some config json file or something?
 	internalConfig := internalParameters{
-		greediness:       0,
-		selfishness:      0,
-		fairness:         0,
-		collaboration:    0,
-		riskTaking:       0,
-		maxPardonTime:    3,
-		maxTierToPardon:  shared.SanctionTier3,
-		minTrustToPardon: 0.6,
+		greediness:                   0,
+		selfishness:                  0,
+		fairness:                     0,
+		collaboration:                0,
+		riskTaking:                   0,
+		maxPardonTime:                3,
```

Comment:
Just check with @MikeIsMike if he knows if there are any issues when initialising to 0 and not 0.5 for example

Reply:
I think I took it from @MikeIsMike